xlators/cluster/afr/src/afr-self-heal-entry.c

```diff
@@ -181,16 +181,20 @@ afr_selfheal_entry_delete(xlator_t *this, inode_t *dir, const char *name,
 static int
 afr_new_entry_mark_status(call_frame_t *frame, loc_t *loc,
                           struct afr_reply *lookup_replies,
-                          unsigned char *sources, int source, int dst)
+                          unsigned char *sources, int source, int dst,
+                          gf_boolean_t *dst_hardlink)
 {
     xlator_t *this = frame->this;
     afr_private_t *priv = this->private;
+    struct iatt *iatt = NULL;
     int pending = 0;
     int metadata_idx = 0;
     int idx = -1;
     int ret = 0;
     int i = 0;
 
+    iatt = &lookup_replies[source].poststat;
```

Comment:
The variable assignment after validating source value at line 198.

Reply:
This is not fixed yet.